commons-packet/commons-packet-manager/src/main/java/io/mosip/commons/packet/keeper/PacketKeeper.java

```diff
@@ -126,7 +129,9 @@ public boolean checkIntegrity(PacketInfo packetInfo, byte[] encryptedSubPacket)
      */
     public boolean checkSignature(Packet packet, byte[] encryptedSubPacket) throws NoSuchAlgorithmException {
     	String machineId = packet.getPacketInfo().getId().substring(centerIdLength, centerIdLength+machineIdLength);
-        boolean result = getCryptoService().verify(machineId, packet.getPacket(), CryptoUtil.decodeBase64(packet.getPacketInfo().getSignature()));
+        boolean result = isSignatureReqd ? 

```

Comment:
Invert this check based on the new naming

Reply:
inverted this check, with new naming